contracts/AssetTransfer.sol

```diff
@@ -47,97 +40,99 @@ contract AssetTransfer is WorkbenchBase('AssetTransfer', 'AssetTransfer')
     address public InstanceAppraiser;
 
     constructor (string memory description, uint256 price) public {
+        ApplicationName = "AssetTransfer";
+        WorkflowName = "AssetTransfer";
         InstanceOwner = msg.sender;
         AskingPrice = price;
         Description = description;
         State = StateType.Active;
-        ContractCreated();
+        LogContractCreated();
     }
 
-    function Terminate() public
-    {
+    function Terminate() public {
         if (InstanceOwner != msg.sender) {
-            revert('The contract can only be terminated by the owner');
+            revert("The contract can only be terminated by the owner");
         }
 
         State = StateType.Terminated;
-        ContractUpdated('Terminate');
+        LogContractUpdated('Terminate');
     }
 
     function Modify(string calldata description, uint256 price) external {
         if (State != StateType.Active) {
-            revert('Modify function can only be called when in Active state');
+            revert("Modify function can only be called when in Active state");
         }
 
         if (InstanceOwner != msg.sender) {
-            revert('Modify function can only be called by the owner');
+            revert("Modify function can only be called by the owner");
         }
 
         Description = description;
         AskingPrice = price;
-        ContractUpdated('Modify');
+        LogContractUpdated('Modify');
     }
 
     function MakeOffer(address inspector, address appraiser, uint256 offerPrice) external {
         if (inspector == address(0x000) || appraiser == address(0x000) || offerPrice == 0) {
-            revert('MakeOffer function need to have a valid inspector/appraiser address and an offerPrice > 0');
+            revert("MakeOffer function need to have a valid inspector/appraiser address and an offerPrice > 0");
```

Comment:
Small style thing but revert string should be < 76 characters.

Reply:
fixed